LICENSE

```diff
@@ -1,9 +1,9 @@
+The MIT License (MIT)
```

Comment:
```suggestion
MIT License
```

At least based on https://choosealicense.com/licenses/mit/, I think this should resolve your issue?

Reply:
Yeah it should do thanks, I initially put this extra bit in to follow suit with other golang libs that use MIT but agree this should be absolutely fine.